contracts/Token.sol

```diff
@@ -0,0 +1,57 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+contract Token {
+  string public name = "Leocode Token";
+  string public symbol = "LEO";
+  uint8 public decimals = 18;
+  uint256 public totalSupply = 100000000000000000000000;
```

Comment:
To warto jednak upraszczać, bo ludzie mają problem z czytaniem kodu z taką liczbą cyfr :D Łatwo się jebnąć.
A że Solidity ma do tego operator, to zapis jest banalny. (`100_000 * (10 ** decimals())`)